tests/snippets/test_frame.py

```diff
@@ -708,3 +708,22 @@ def test_types_dot() -> None:
     df6: pd.DataFrame = df1.dot(np_array)
     df7: pd.Series = df1 @ s1
     df8: pd.Series = df1.dot(s1)
+
+
+def test_to_df() -> None:
+    # https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.to_dict.html
+
+    df = pd.DataFrame([[0, 1]], columns=["A", "B"])
+
+    # If just records, it should return a list
+    records: List[Mapping[Any, Any]] = df.to_dict(orient="records")
```

Comment:
Why `Mapping[Any, Any]` if the method declarations have it at `Mapping[Column, Any]`?

Reply:
That is a very fair point! Adding now....